src/proc/colorizer.h

```diff
@@ -45,13 +32,13 @@ namespace librealsense {
 
         color_map() {}
 
-        float3 get(float value) const
-        {
+        inline float3 get(float value) const;
```

Comment:
I'm not sure this is necessary
(why not just keep the function in place? )

Reply:
I'm reverting it to the header, but I also see that the complete color_map implemented twice (c&p)  in colorizer.h and rendering.h - I'm adding it to the refactoring list.